benefits/eligibility/templates/eligibility/includes/modal--contactless.html

```diff
@@ -13,11 +13,7 @@ <h2 id="contactless-modal" class="me-4 me-md-0">{% translate "Learn more about c
     </p>
     <p class="pt-4">{% translate "The contactless symbol is four curved lines, like this:" %}</p>
     <p class="contactless-symbol pt-3">
-      <img class="icon mx-auto d-block"
-           width="30"
-           height="37.5"
-           src="{% static 'img/icon/contactless.svg' %}"
-           alt="{% translate "Four curved lines on contactless-enabled cards" context "image alt text" %}" />
+      <img class="icon mx-auto d-block" width="30" height="37.5" src="{% static 'img/icon/contactless.svg' %}" alt="" />
```

Comment:
Need to add the `alt` text back here per above conversation

Reply:
Added back: https://github.com/cal-itp/benefits/pull/1654/commits/bf860bc0f9fd03d52c51cc4357fd771eee6d9114